packages/dashboard/src/customization/propertiesSections/lineAndScatterStyleSettings/section.tsx

```diff
@@ -220,7 +219,7 @@ const RenderLineAndScatterStyleSettingsSection = ({
   };
 
   return (
-    <SpaceBetween size='s' direction='vertical'>
+    <>
```

Comment:
you can completely remove this ?

Reply:
Need to wrap everything under one div/fragment. Removed the SpaceBetween because navita's design shows that we dont need to have a gap between each drop down.